packages/shop/src/containers/banner-ad/BannerAd.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import 'styled-components/macro';
+
+import useBannerAd from '../../hooks/use-banner-ad';
+import { fireTrackingEvents } from '../../common/utils/tracker';
+import Banner from '../../components/banner';
+
+type Props = {
+  className?: string;
+  inventoryId: string;
+  items?: string[];
+  searchQuery?: string;
+  width: number;
+  height: number;
+};
+
+export const BannerAd = (props: Props) => {
+  const { className, inventoryId, items, searchQuery, width, height } = props;
+
+  const router = useRouter();
+
+  const { isLoading, data, isError } = useBannerAd({
+    inventoryId,
+    items,
+    searchQuery,
+  });
+
+  const { bannerAd } = data || {};
+
+  const handleBannerClick = () => {
+    if (!bannerAd) return;
+    const { clickTrackers } = bannerAd;
+
+    if (clickTrackers) {
+      fireTrackingEvents(clickTrackers);
+    }
+
+    // TODO: enable navigation
+    // router.push(`/special-offer?requestId=${bannerAd.requestId}`);
+  };
+
+  const handleBannerVisible = () => {
+    console.log('handleBannerVisible', { bannerAd });
```

Comment:
Please remove this line.

Reply:
Fixed in https://github.com/moloco-rmp/demo-store/pull/20/commits/1779be441982e3a1a8369bf79f03f41e98122d5f